modules/ui/src/main/scala/lucuma/ui/forms/FormInputEV.scala

```diff
@@ -94,34 +105,87 @@ object FormInputEV {
       props.onValidChange(validated.isValid) >> cb(validated)
     }
 
+    def getInputElement(e: html.Element): CallbackOption[html.Input] = {
+      def childChain(ele: html.Element): Chain[html.Element] =
+        Chain.fromSeq(
+          (0 until ele.childElementCount).map(i => ele.children.item(i).asInstanceOf[html.Element])
+        )
+
+      def findWithClass(ele: html.Element, acc: Chain[html.Element]): Chain[html.Element] = {
+        val newAcc   = if (ele.className.contains("input")) acc :+ ele else acc
```

Comment:
Could we use the ref here?

Reply:
I'm not sure what you are asking. The Ref points to a div, and I need to search down to find the div that contains only the `input` control. There is nothing to identify the `input` control itself. Are you asking if I could use the `outerRef` directly in `getInputElement` instead of in the functions that call it? (`getCursor` and `setCursor`).